src/olympia/migrations/1107-yara.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE `yara_results` (
+    `id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY,
+    `created` datetime(6) NOT NULL,
+    `modified` datetime(6) NOT NULL,
+    `upload_id` int(11),
+    `version_id` integer unsigned,
+    `matches` longtext,
+
+    FOREIGN KEY (`upload_id`) REFERENCES `file_uploads` (`id`) ON DELETE SET NULL,
+    FOREIGN KEY (`version_id`) REFERENCES `versions` (`id`)
```

Comment:
This sql is missing the named `CONSTRAINT` we usually have.  Fwiw, I find the easiest way to get the exact same sql as django would generate is to run the tests with `--create-db` then grab the sql from the `olympia_test` database with `SHOW CREATE TABLE xxx`.

Reply:
Thanks!